src/auto/enums.rs

```diff
@@ -1729,6 +1729,86 @@ impl SetValue for EventSequenceState {
     }
 }
 
+#[derive(Debug, Eq, PartialEq, Ord, PartialOrd, Hash, Clone, Copy)]
+#[non_exhaustive]
+pub enum ExpanderStyle {
+    Collapsed,
+    SemiCollapsed,
+    SemiExpanded,
+    Expanded,
+    #[doc(hidden)]
+    __Unknown(i32),
+}
+
+impl fmt::Display for ExpanderStyle {
```

Comment:
This does not seem to be used in any API we have, so just having the enum generated wouldn't provide anything useful.